File: lista 2/4.py
a, b, c = map(int, input().split())

total = a + b + c

if total > 80 or total < 20 or a > 50 or b > 50 or c > 50:
    print("Shrek vai dar o urro!")
else:
    print("Shrek ficou calmo.")

File: lista 2/4.cpp
#include <iostream>

using namespace std;

int main() {
    int a, b, c;
    cin >> a >> b >> c;

    int total = a + b + c;

    if (total > 80 || total < 20 || a > 50 || b > 50 || c > 50) {
        cout << "Shrek vai dar o urro!" << endl;
    } else {
        cout << "Shrek ficou calmo." << endl;
    }

    return 0;
}